index.js

```diff
@@ -39,6 +39,7 @@ function RegClient (config) {
   if (typeof this.config.retry.factor !== 'number') this.config.retry.factor = 10
   if (typeof this.config.retry.minTimeout !== 'number') this.config.retry.minTimeout = 10000
   if (typeof this.config.retry.maxTimeout !== 'number') this.config.retry.maxTimeout = 60000
+  if (typeof this.config.maxSockets !== 'number') this.config.maxSockets = 50
```

Comment:
The number 50 is pretty arbitrary but seemed to perform okay. Maybe the default should be lower or higher though. I've tried a few different values with a big shrinkwrap install and it didn't seem to make much difference past 20.